packages/openactive-integration-tests/test/templates/b-req.js

```diff
@@ -23,6 +23,7 @@ const { createPaymentPart, isPaidOpportunity, isPaymentAvailable, addOrderItemIn
  * }} AccessPassItem
  *
  * @typedef {{
+ *   orderType: 'Order' | 'OrderProposal',
```

Comment:
p-req.js is gone. Instead, P requests just use b-req.js templates with `orderType` set to `OrderProposal`. Seems to work 👍